PBTutorialManager/Pods/Local Podspecs/PBTutorialManager.podspec.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "PBTutorialManager",
+  "platforms": {
+    "ios": "8.0"
+  },
+  "version": "1.0.4",
```

Comment:
I went for a slightly higher version number as there's quite a lot of changes and new features. I think I've done all the changes you requested, but let me know if anything is missing or if you spot anything else.

I really like the your library, it was just a pity some swift syntax changes broke it. In case your wondering what I'm using it for, it'll be in the next version of my skydiving app. https://itunes.apple.com/us/app/skydive-logger/id971853125?mt=8

Reply:
Yes I saw that. No problem for me with 1.1.0 instead of 1.0.5. It makes more sense indeed.
I just merged your PR and update podspec file to update the pod library. 
**It is now available !** `pod 'PBTutorialManager'` should now target 1.1.0 by default ;)

Thanks for your work. Your app looks great and very interesting thanks to let me know about this. Feel free to add a reference to your app in the README if you want to.

Have a nice day